aip_common_sensor_launch/launch/nebula_node_container.launch.py

```diff
@@ -336,7 +433,26 @@ def add_launch_arg(name: str, default_value=None, description=None):
     add_launch_arg("advanced_diagnostics", "false")
     add_launch_arg("use_multithread", "False", "use multithread")
     add_launch_arg("use_intra_process", "False", "use ROS 2 component container communication")
-    add_launch_arg("lidar_container_name", "nebula_node_container")
+    add_launch_arg(
+        "lidar_container_name",
+        "nebula_node_container",
+        "Name of the new container to be created when use_shared_container is false",
+    )
+    add_launch_arg(
+        "shared_container_name",
+        "pointcloud_container",
+        "Name of the existing container to be used when use_shared_container is true",
+    )
```

Comment:
These two should probably be combined for clarity. They can never validly be set at the same time and the shared container vs. new container aspect is already handled by the use_shared_container flag.

Reply:
Addressed in [fc2b7fd](https://github.com/tier4/aip_launcher/pull/348/commits/fc2b7fd2992092e5b43812fb42015bea96ec9492)
The consequence is that all containers will be called `pointcloud_container` which will need tooling that relied on this to be updated (not sure if the container name was ever used though)